app/test_main.py

```diff
@@ -1,3 +1,19 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, result",
+    [
+        (0, 0, [0, 0]),  # test should return zero age
```

Comment:
There is no need to duplicate test content with comments, remove comments since data is self-explanatory